.github/workflows/nightly.yml

```diff
@@ -150,6 +150,9 @@ jobs:
       - name: "Set Vector build description"
         run: ./scripts/set-build-description.sh
         shell: bash
+      - name: "Set release flags"
```

Comment:
Actually, should this be set on all of the `build-*` jobs? You could potentially add it to the top-level `env:` as well.

Reply:
This is a little confusing/tricky: we call `release-flags.sh` in `prepare.sh`, but `prepare.sh` is only used for Linux-based jobs... so that's why we call it manually for macOS and Window.

I'm going to file a ticket to come up with a better/simpler approach for all of this... ideally some sort of all-in-one script that figures out the current architecture, etc, so that we can safely run a single script prior to _any_ job, and it does the Right Thing (tm).

Reply:
Aha, doh, missed that. Thanks!